app/Jobs/DeploymentQueueJob.php

```diff
@@ -57,13 +58,30 @@ public function handle()
             $deployOutput = new DeployOutputs();
             $deployOutput->output = $line.PHP_EOL;
             $deploy->outputs()->save($deployOutput);
+
+            Redis::publish('output-channel', json_encode([
```

Comment:
you shouldn't use Redis directly, Event broadcast has a redis driver that does that for you... You basically dispatch jobs